src/mod_zone_difficulty_scripts.cpp

```diff
@@ -27,30 +33,71 @@ void ZoneDifficulty::LoadMapDifficultySettings()
         return;
     }
 
+    sZoneDifficulty->Rewards.clear();
+
     // Default values for when there is no entry in the db for duels (index 0xFFFFFFFF)
-    ZoneDifficultyInfo[DUEL_INDEX][0].HealingNerfPct = 1;
-    ZoneDifficultyInfo[DUEL_INDEX][0].AbsorbNerfPct = 1;
-    ZoneDifficultyInfo[DUEL_INDEX][0].MeleeDamageBuffPct = 1;
-    ZoneDifficultyInfo[DUEL_INDEX][0].SpellDamageBuffPct = 1;
+    NerfInfo[DUEL_INDEX][0].HealingNerfPct = 1;
+    NerfInfo[DUEL_INDEX][0].AbsorbNerfPct = 1;
+    NerfInfo[DUEL_INDEX][0].MeleeDamageBuffPct = 1;
+    NerfInfo[DUEL_INDEX][0].SpellDamageBuffPct = 1;
+
+    // Heroic Quest -> MapId Translation
+    HeroicTBCQuestMapList[542] = 11362;
+    HeroicTBCQuestMapList[543] = 11354;
+    HeroicTBCQuestMapList[547] = 11368;
+    HeroicTBCQuestMapList[546] = 11369;
+    HeroicTBCQuestMapList[557] = 11373;
+    HeroicTBCQuestMapList[558] = 11374;
+    HeroicTBCQuestMapList[560] = 11378;
+    HeroicTBCQuestMapList[556] = 11372;
+    HeroicTBCQuestMapList[585] = 11499;
+    HeroicTBCQuestMapList[555] = 11375;
+    HeroicTBCQuestMapList[540] = 11363;
+    HeroicTBCQuestMapList[552] = 11388;
+    HeroicTBCQuestMapList[269] = 11382;
+    HeroicTBCQuestMapList[553] = 11384;
+    HeroicTBCQuestMapList[554] = 11386;
+    HeroicTBCQuestMapList[545] = 11370;
```

Comment:
This will probably never be changed, but for eas I would add a comment at the end of every line which Map we are talking about

`HeroicTBCQuestMapList[mapId] = 11373; // Stockades`